src/store/traction/ont/actions.js

```diff
@@ -1,17 +1,433 @@
 import { handleResponse } from '@/api/ResponseHelper'
```

Comment:
I think it is right to try to move stuff to a helper file but maybe not now?
Some of these files could do with some standard documentation.
Also this file was so large it didn't load the diff so probably means we need to split it at some point.

Reply:
Agreed it is large, ill change the comment to a todo and add documentation 👍